### DISCUSSION TITLE
FT8CN for RCFORB Android client (www.remotehams.com)

### DISCUSSION THREAD

**Discussion by Ringo1992** [N0BOY/FT8CN#79] [Ideas]
Dear BG7YOZ, thousands of HAM Radio Operators worldwide are using the Android RCFORB client app for remote operation of their amateur radio stations. It would be great if there would be a possibility in FT8CN to directly transfer audio between these both apps within Android and to set RCFORB to transmit/receive mode directly from FT8CN (same like pressing TXD button in RCFORB). In a first step no CAT frequency control of the RCFORB client is necessary, but this could be an option for further developments. These new features would allow me to use your great FT8CN app at all places around the world by using my station at home from every location abroad. 
73 de Rainer, DL1RBR

**Comment by bg7yoz** [comment]
hi, @Ringo1992 

你的想法很棒！只是我对RCFORB缺乏了解，暂时无法实现:-P

bg7yoz

**Reply by Ringo1992** [reply]
In a first step the simple solution could be to generally stream all audio output of Android to the input of FT8CN and stream all audio output of FT8CN to the standard Android microphone/headset input so that there is no audio hearable; RX/TX switching could be done manually or by VOX then. At the moment it already works by I have to listen to the FT8 sound all the time, which is a bit annoying; thanks in advance

**Comment by wiadziu** [comment]
this is what we need, at the moment I tried the same principle as @Ringo1992  did, pressing PTT manually and listening to the beeping as the decoded FT8 sound, I don't know how much I could help but I somehow know the author of RCforb in the Android version, for this Over time, it is also clear that the application has stagnated and is not being developed with new functions...

In my imagination, I also see an option for the FT8CN program to run, for example, on a PC as a server, e.g. through a browser, as now there are various functions such as export, statistics impot, etc.

while on Android it works great via CAT with my TS-480 radio and interface (USB-C) with audio and 2 COM ports

It would also be good to develop the conect via Network option, as is the case with ICom and Flex, I don't know how it works, but maybe I'll try it in the future

or maybe someone found an option to, for example, connect it on Android using a virtual cable and somehow activate VOX ?

**Comment by ra3trm** [comment]
Sorry!
Add the CAT YAESU FT-747GX to the FT8CN program.
Why is there a YAESU FT-747GX in WJTDX, but not in FT8CN ?
Not everyone has the latest transceivers!

THANKS!

**Comment by ra3trm** [comment]
嗨！
旧设备不应该被遗忘。
不是每个人都很富有。 我不是UA3REO。
请添加：YAESU，ICOM，KENWOOD，BARRETT old models
